shared/src/main/scala/amf/client/remod/AMFEnvironment.scala

```diff
@@ -56,6 +56,7 @@ private[amf] case class AMFEnvironment(override val resolvers: AMFResolvers,
   type Self = AMFEnvironment
 
   def withParsingOptions(parsingOptions: ParsingOptions): AMFEnvironment = this.copy(options = options.copy(parsingOptions = parsingOptions))
+  def withRenderingOptions(renderingOptions: RenderOptions): AMFEnvironment = this.copy(options = options.copy(renderingOptions = renderingOptions))
```

Comment:
probably leave `withRenderOptions` to have "coherence" between the options object name and the setter.

Reply:
good catch